internal/tools/tools.go

```diff
@@ -10,11 +10,11 @@ import (
 	_ "github.com/Khan/genqlient"
 	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	_ "github.com/goreleaser/goreleaser"
-	_ "github.com/open-telemetry/opentelemetry-collector-contrib/cmd/mdatagen"
 	_ "github.com/securego/gosec/v2/cmd/gosec"
 	_ "go.opentelemetry.io/build-tools/crosslink"
 	_ "go.opentelemetry.io/build-tools/multimod"
 	_ "go.opentelemetry.io/collector/cmd/builder"
+	_ "go.opentelemetry.io/collector/cmd/mdatagen"
```

Comment:
`mdatagen` was migrated to the core collector repo in `v0.97.0` which module name is `go.opentelemetry.io/*`